Source/Plugin.BLE.Android/Adapter.cs

```diff
@@ -154,12 +156,62 @@ protected override void StopScanNative()
             }
         }
 
-        protected override Task ConnectToDeviceNativeAsync(IDevice device, bool autoconnect, CancellationToken cancellationToken)
+        protected override Task ConnectToDeviceNativeAsync(IDevice device, bool autoconnect, CancellationToken cancellationToken, bool forceBleTransport)
         {
             AddToDeviceOperationRegistry(device);
-            ((BluetoothDevice)device.NativeDevice).ConnectGatt(Application.Context, autoconnect, _gattCallback);
+
+            #region isLeBrErdCapableHost : Possible default decision for Simultaneous LE and BR/EDR to Same Device Capable -> DualMode
+
+            //sometimes with dual mode devices, we have to use the LE transport in the connectGatt method.
+            //otherwise the device will not respond to a create connection request (legacy)
+            //the device will only respond to a create LE connection
+            //by default the transport mode is AUTO and in most case Android will decide to use BR/EDR connection (legacy)
+            //the following is a proposition of how handle it by default.
+            //if the device advertise for the 'Simultaneous LE and BR/EDR to Same Device Capable' capability (DualMode)
+            //the default transport mode should be LE in most of context
+            //For now, just add a boolean to force BLE transport in connectToDevice methods
+
+            //bool isLeBrErdCapableHost = nativeDevice.Type == BluetoothDeviceType.Dual;
```

Comment:
Just I was wondering, if ALL Dual devices work as mine, if it was better to automatically use the BLE transport parameter. So that's why I'm not sure of this feature and in doubt I prefered to implement a boolean to explicitly set the behavior. If someone knows better than me on the subject, I think it's an relevant question.